simulariumio/data_objects/model_meta_data.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from typing import Dict, Any
+import logging
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class ModelMetaData:
+    model_title: str
+    model_version: float
+    model_authors: str
+    model_description: str
+    doi: str
+    source_code_url: str
+    source_code_license_url: str
+    input_data_url: str
+    raw_output_data_url: str
+
+    def __init__(
+        self,
+        model_title: str = None,
+        model_version: float = -1.0,
```

Comment:
version should be string?   Unless you want to enforce your own versioning system, if this is a user-defined value, people put all kinds of strings in their version numbers like alpha, release candidate etc etc)

Reply:
on the other hand, a single number does keep it simple.. just want to make sure it is fitting the intent

Reply:
yeah I think a string is better actually, forgot about that